ml/README.md

```diff
@@ -44,6 +53,10 @@ The model achieves an accuracy of **96.6%** on the validation dataset.
 
 The ratio of training set to validation set is **1000 : 100**.
 
+## How to train the model
+
+To train your model, open `ModelTrainer.ipynb` in jupyter notebook to train your model. If you don't have jupyter notebook installed, you can convert .ipynb to .py file by running this command in your terminal `ipython nbconvert --to python ModelTrainer.ipynb`. Now you can just run `python3 ModelTrainer.py`.
```

Comment:
How does `nbconvert` work? Does it generate a new python file?

Reply:
Yes, nbconvert converts the ipynb file into a python file.